### PR TITLE
IEP008: IntelMQ Data Format: Constituency field

### PR DESCRIPTION
**Pull request by kamil-certat**
Proposal of adding a constituency field to the IDF.

### CODE REVIEW

File: 008/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+and finally define a way to correlate them.
+
+## Proposal
+
+Because of the simplicity, I've chosen to propose adding a simple text field in the IDF, defined as:
+
+```
+"constituency": {
+    "description": "Internal identifier for multi-constituency setup",
+    "length": 10,
```
Comment (sebix) [inline 008/README.md]: Why limit it to a certain length? How do you know that the constitutency specifiers of IntelMQ operators, active in various different legislative, linguistic, and organizational environments are all shorter or equal to 10 characters?

**Comment by sebix** [comment]
You may start a thread on the mailing lists to get a discussion and feedback (and afterward, hopefully, a conclusion ;) ).

**Comment by kamil-certat** [comment]
> You may start a thread on the mailing lists 

Hmm, I sent the email the same time I've opened the PR as ready. Should I send it somewhere else?

**Comment by sebix** [comment]
> > You may start a thread on the mailing lists
> 
> Hmm, I sent the email the same time I've opened the PR as ready. Should I send it somewhere else?

Duh, sorry. Looks like I missed that (as everyone else?). Now you're hitting the same problem as every time when we ask for input from the community.

File: 008/README.md (hunk 1 above)
Comment (kamil-certat) [reply 008/README.md]: My idea was about putting here just an identifier. But you are right, it's better to leave it to the operator. I remove the limit

**Pull request merged**